projects/igniteui-angular/src/lib/grids/grid/grid-mrl-keyboard-nav.spec.ts

```diff
@@ -1,2903 +1,3104 @@
-﻿import { Component, ViewChild } from '@angular/core';
-import { async, TestBed } from '@angular/core/testing';
+﻿import { Component, ViewChild, Type, DebugElement, NgZone } from '@angular/core';
+import { async, TestBed, ComponentFixture } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
 import { IgxGridModule, IgxGridCellComponent, IGridCellEventArgs } from './index';
 import { IgxGridComponent } from './grid.component';
 import { SampleTestData } from '../../test-utils/sample-test-data.spec';
 import { configureTestSuite } from '../../test-utils/configure-suite';
 import { wait, UIInteractions } from '../../test-utils/ui-interactions.spec';
-import { setupGridScrollDetection } from '../../test-utils/helper-utils.spec';
+import { setupGridScrollDetection, TestNgZone } from '../../test-utils/helper-utils.spec';
 import { SortingDirection } from '../../data-operations/sorting-expression.interface';
 import { DefaultSortingStrategy } from '../../data-operations/sorting-strategy';
+import { IgxGridGroupByRowComponent } from './groupby-row.component';
+import { GridFunctions } from '../../test-utils/grid-functions.spec';
 
 const DEBOUNCETIME = 30;
 const CELL_CSS_CLASS = '.igx-grid__td';
 const ROW_CSS_CLASS = '.igx-grid__tr';
 const CELL_BLOCK = '.igx-grid__mrl-block';
+const focusEvent = new FocusEvent('focus');
 
 describe('IgxGrid Multi Row Layout - Keyboard navigation #grid', () => {
-    configureTestSuite();
+    let fix: ComponentFixture<ColumnLayoutTestComponent>;
+    let zone: TestNgZone;
+
+    describe('Without scrolling', () => {
+        configureTestSuite();
+        beforeAll(async(() => {
```

Comment:
As all the describes use same component and imports you can move it under base describe.